OpenRA.Mods.Common/Traits/Contrail.cs

```diff
@@ -71,6 +71,12 @@ IEnumerable<IRenderable> IRender.Render(Actor self, WorldRenderer wr)
 			return new IRenderable[] { trail };
 		}
 
+		Rectangle IRender.AutoRenderBounds(Actor self)
+		{
+			// Actor.RenderBounds unions all non-empty sprite bounds, so this wouldn't have an effect on the final bounds anyway as RenderSprites/RenderBounds take care of that
+			return Rectangle.Empty;
```

Comment:
Likewise here, if we want the contrails to count towards the ScreenMap bounds (which we do... in the long term at least... if we don't want them to be cut off when the actor's body is outside the screen).